modules/meeting/app/components/meetings/sidebar/state_component.sass

```diff
@@ -0,0 +1,9 @@
+@import 'helpers'
+
+@media screen and (max-width: $breakpoint-sm)
+  .op-meeting-sidebar-state
+    flex-direction: row !important
+    justify-content: space-between
+    align-items: center
+    &-edit
+      margin-top: 0 !important
```

Comment:
Same as before: All of this could be shown with `system_arguments`. I am fine with keeping the flex defintion, but please remove the margin, as well as the extra class for it.